source/_integrations/signal_messenger.markdown

```diff
@@ -90,3 +90,19 @@ action:
       attachments:
         - "/tmp/surveillance_camera.jpg"
 ```
+
+### Text message with an attachment from a URL
+
+Note there is a 50mb size limit for attachments retrieved via URLs. You can set `verifySsl` to `False` to ignore SSL errors (default `True`)
+
+```yaml
+...
+action:
+  service: notify.NOTIFIER_NAME
+  data:
+    message: "Person detected on Front Camera!"
+    data:
+      verifySsl: False
```

Comment:
`verifySsl` isn't following our standards; it should be:

```suggestion
      verify_ssl: false
```

Additionally, I'm confused why Home Assistant needs to verify SSL, to begin with?